application/notifications/messages.go

```diff
@@ -9,4 +9,5 @@ type Message struct {
 	ToName    string
 	ToEmail   string
 	ToPhone   string
+	Subject   string
```

Comment:
This field isn't read anywhere. Looks like `(e *EmailNotifier) Send` function needs to pass it on to the email service.

Reply:
Good catch.